src/components/TitleTransferPanel/TitleTransferPanelContent/TitleTransferPanelContent.tsx

```diff
@@ -12,15 +12,15 @@ interface TitleTransferPanelContentProps {
 export const TitleTransferPanelContent = ({ document }: TitleTransferPanelContentProps) => {
   const [tokenOwnerAddress, setTokenOwnerAddress] = useState("");
   const { holderAddress, beneficiaryAddress } = useEscrowContractUsers({ escrowContractAddress: tokenOwnerAddress });
-  const fetchTokenOwner = async () => {
-    try {
+
+  useEffect(() => {
+    const fetchTokenOwner = async () => {
```

Comment:
You can put this outside the useEffect and only call it with fetchTokenOwner inside the useEffect

Reply:
oh i tried that but lint gimme this hence i put inside.

![Screenshot 2020-03-26 at 2 02 30 PM](https://user-images.githubusercontent.com/4774314/77615517-87931000-6f6a-11ea-963d-0d727e631497.png)

Reply:
haha make the function signature of fetchTokenOwner to take the document in as a param. 

const fetchTokenOwner = async (document) => {
  const address = await getTokenOwner({ document });
  setTokenOwnerAddress(address);
}